### PR TITLE
DOC-4250 describe ldap synchronization job configuration params

### PR DESCRIPTION
**Pull request by madhifallah**

### CODE REVIEW

File: docs/LDAP.rst [hunk 1]
```diff
@@ -1057,3 +1057,47 @@ To resolve this, add an ACL (Access Control List) rule in the ``slapd.conf`` fil
 .. |image0| image:: images/LDAP/ldap_integration.png   
 .. |image1| image:: images/LDAP/ldap_user.png    
 .. |image2| image:: images/LDAP/GroupsDNs.png   
+
+=================================================
+Organization externalStore jobs configuration
+=================================================
+
+Organization import and synchronization is performed by IDMExternalStoreImportService.
```
Comment (thomasdelhomenie) [inline docs/LDAP.rst]: We should at first explain what is the goal and the principle of this import.

File: docs/LDAP.rst [hunk 2]
```diff
@@ -1057,3 +1057,47 @@ To resolve this, add an ACL (Access Control List) rule in the ``slapd.conf`` fil
...
+				<description>Cron expression used to schedule the job that will delete periodically data from internal store that has been deleted from external store (Default value = every day at 23:59 PM)</description>
+				<value>${exo.idm.externalStore.delete.cronExpression:0 59 23 ? * *}</value>
+			</value-param>
+			<value-param>
+				<name>exo.idm.externalStore.queue.processing.cronExpression</name>
+				<description>Cron expression used to schedule the job that will process periodically data injected in queue (Default value = every minute)</description>
+				<value>${exo.idm.externalStore.queue.processing.cronExpression:0 */1 * ? * *}</value>
+			</value-param>
+		</init-params>
+	</component>
```
Comment (thomasdelhomenie) [inline docs/LDAP.rst]: I don't think it is useful to put the XML configuration. An admin only needs the configuration properties.

File: docs/LDAP.rst [hunk 3]
```diff
@@ -1057,3 +1057,47 @@ To resolve this, add an ACL (Access Control List) rule in the ``slapd.conf`` fil
...
+
+
+**IDMExternalStoreImportService** customizable parameters are :
```
Comment (thomasdelhomenie) [inline docs/LDAP.rst]:
```suggestion
The available configuration parameters are :
```

File: docs/LDAP.rst [hunk 4]
```diff
@@ -1057,3 +1057,47 @@ To resolve this, add an ACL (Access Control List) rule in the ``slapd.conf`` fil
...
+
+**exo.idm.externalStore.import.cronExpression**
+Cron expression used to schedule the job that will import periodically data from external store (Default value = every ten minutes)
```
Comment (thomasdelhomenie) [inline docs/LDAP.rst]:
```suggestion
Cron expression used to schedule the job that will import periodically data from external store. Defaults to `0 */10 * ? * *` (every ten minutes).
```

File: docs/LDAP.rst [hunk 5]
```diff
@@ -1057,3 +1057,47 @@ To resolve this, add an ACL (Access Control List) rule in the ``slapd.conf`` fil
...
+
+**exo.idm.externalStore.delete.cronExpression**
+Cron expression used to schedule the job that will delete periodically data from internal store that has been deleted from external store (Default value = every day at 23:59 PM)
```
Comment (thomasdelhomenie) [inline docs/LDAP.rst]:
```suggestion
Cron expression used to schedule the job that will delete periodically data from internal store that has been deleted from external store. Defaults to `0 59 23 ? * *` (every day at 23:59 PM).
```

File: docs/LDAP.rst [hunk 6]
```diff
@@ -1057,3 +1057,47 @@ To resolve this, add an ACL (Access Control List) rule in the ``slapd.conf`` fil
...
+
+**exo.idm.externalStore.queue.processing.cronExpression**
+Cron expression used to schedule the job that will process periodically data injected in queue (Default value = every minute)
```
Comment (thomasdelhomenie) [inline docs/LDAP.rst]:
```suggestion
Cron expression used to schedule the job that will process periodically data injected in queue. Defaults to `0 */1 * ? * *` (every minute).
```

**Review by thomasdelhomenie — DISMISSED** [review]

File: docs/LDAP.rst (hunk 6 above)
Comment (madhifallah) [reply docs/LDAP.rst]: done

File: docs/LDAP.rst (hunk 5 above)
Comment (madhifallah) [reply docs/LDAP.rst]: done

File: docs/LDAP.rst (hunk 4 above)
Comment (madhifallah) [reply docs/LDAP.rst]: done

File: docs/LDAP.rst (hunk 3 above)
Comment (madhifallah) [reply docs/LDAP.rst]: done

File: docs/LDAP.rst (hunk 2 above)
Comment (madhifallah) [reply docs/LDAP.rst]: removed

File: docs/LDAP.rst (hunk 1 above)
Comment (madhifallah) [reply docs/LDAP.rst]: goal and  principle of the import were added.

**Review by mzorai — APPROVED** [review]

**Pull request merged**